<!-- Launch file to store the names of dVRK topics -->
<launch>
	<group ns="robot_interface_topics">
	<param name="namespace" type="string" value="/irob_autosurg/robot_interface" />
	
	<param name="position_cartesian_current" type="string"
								 value="position_cartesian_current" />
	
	</group>
	
</launch>
